src/Context/CommonContext.php

```diff
@@ -391,18 +391,19 @@ public function iShouldNotSeeElementByCss($selector): void
      */
     public function thenIShouldSeePage(string $url): void
     {
+        $current_url = $this->getSession()->getCurrentUrl();
         $windowNames = $this->getSession()->getWindowNames();
         if (count($windowNames) > 1) {
             $this->switchToNewTab($windowNames);
+            $current_url = $this->getSession()->getCurrentUrl();
+            $this->getSession()->executeScript('window.close()');
```

Comment:
This is introducing a new behaviour and closing the page/tab, which it was not doing before and it is not clear on this methods name or gherkin language definition. If it is a new behaviour, it should be a separate method or it should be made clear on definition of this content implementation.

Also: according to PSR standards, please use `$camelCase` variable names.